st2common/tests/unit/test_jinja_render_regex_filters.py

```diff
@@ -68,3 +68,24 @@ def test_filters_regex_search(self):
         actual = env.from_string(template).render({'k1': 'v0.10.1'})
         expected = 'True'
         self.assertEqual(actual, expected)
+
+    def test_filters_regex_substring(self):
+        env = jinja_utils.get_jinja_environment()
+
+        # Normal (match)
+        template = '{{k1 | regex_substring("(127.0.0.1)")}}'
```

Comment:
Good ideas - did this in https://github.com/StackStorm/st2/pull/3482/commits/223b0a33f910feeaaaf6bb3bb732a2b478848e98 (as well as refactored to use `re.findall` instead